spec/integration_spec.rb

```diff
@@ -35,26 +35,10 @@ def phctl(*args)
       end
     end
 
-    it "HtItems creates clusters with ocns in hathifile" do
```

Comment:
Functionality removed; hathifiles database will do all the necessary hathifile loading